Svg.Tests.Win/SvgTextTests.cs

```diff
@@ -142,7 +142,7 @@ public void CanRenderTSpan_DX_WithTransforms()
             using var c = TestHelper.ImageCompare(svgBitmap, pngBitmap);
 
 
-            c.AssertAreSimilar(99f, svgPath);
+            c.AssertAreSimilar(94f, svgPath);
```

Comment:
this is worrysome.
You should not have to reduce the similarity once you implemented a feature. This indicates that you have a regression (i.e. you introduced a render bug)

Actually, when implementing stuff, the similarity should rise!